modules/xfem/doc/content/syntax/XFEM/index.md

```diff
@@ -1 +1,85 @@
-!template load file=stubs/moose_system.md.template name=XFEM syntax=/XFEM
\ No newline at end of file
+# XFEM Action System
+
+!syntax description /XFEM/XFEMAction
+
+## Overview
+
+The `XFEM` block must be supplied to run any MOOSE simulations using XFEM. It provides general
+solution parameters to XFEM. Including this block in the input file serves several functions:
+
+- Causes an algorithm to be run at defined intervals to split the mesh based on a set of evolving defined interfaces.
+- Causes a modified quadrature rule to be used to correctly integrate elements split by these interfaces.
+- Creates objects to generate output related to the XFEM algorithm.
+- Provides a syntax block where various parameters controlling the behavior of XFEM can be specified by the user, including:
+
+  - Provide `qrule` variable to XFEM to determine which quadrature weighting rule to use.
+  - Specify `output_cut_plane` variable to output cut elements to the exodus mesh.
+  - Optional: Provide parameter values for prescribed, incremental crack growth.
+  - Optional: Setup of near tip enrichment parameters.
+  - Optional: Allow users to control the amount of debug information printed during a simulation.
+
+## Constructed MooseObjects
+
+Three of the above functions pass parameters to XFEM for use in setting up relevant options and
+problems: the `qrule` variable, incremental crack growth, and `debug_output_level`. The remaining
+functions have components that replace specific MooseObjects to make input files more concise.
```

Comment:
The remaining parameters result in the creation of various objects, which allows for input files to be more concise.